### PR TITLE
UI control for coverage parameters

### PR DESCRIPTION
**Pull request by myarjunar**
Mar-27-2019 13-43-29

coverage results and map creation parameter will come next

### CODE REVIEW

File: geosys/bridge_api/definitions.py [hunk 1]
```diff
@@ -211,3 +211,59 @@
...
+    INSEASON_NDVI,
+    INSEASONFIELD_AVERAGE_NDVI,
+    INSEASONFIELD_AVERAGE_REVERSE_NDVI,
+    INSEASON_CVI,
+    INSEASON_EVI,
+    INSEASONPARTIAL_NDVI,
+    INSEASONPARTIAL_EVI,
+    INSEASON_GNDVI,
+    COLOR_COMPOSITION
+]
```
Comment (myarjunar) [inline geosys/bridge_api/definitions.py:238]: @AndreLester @timlinux I'm not sure about the map products options. Coverage request with geometry parameter only returns map products included in `INSEASON_MAP_PRODUCTS`. Is this correct?

File: geosys/bridge_api/definitions.py [hunk 2]
```diff
@@ -211,3 +211,59 @@
...
+}
+
+SENTINEL_2 = {
+    'key': 'sentinel-2',
+    'name': 'SENTINEL_2'
+}
+
+SENSORS = [
+    DEIMOS, DMC, LANDSAT_8, RESOURCESAT2, SENTINEL_2
+]
```
Comment (myarjunar) [inline geosys/bridge_api/definitions.py:269]: @AndreLester same with list of sensors. Did this cover all sensors?

File: geosys/bridge_api/definitions.py (hunk 1 above)
Comment (AndreLester) [reply geosys/bridge_api/definitions.py:238]: I will follow this up with the client.

**Comment by AndreLester** [comment]
LGTM

**Pull request merged**